web/src/components/organisms/CommentInput.jsx

```diff
@@ -0,0 +1,15 @@
+import { Label, Textarea } from 'flowbite-react';
+import React from 'react';
+
+const CommentInput = () => {
```

Comment:
It is not used only for comment so let's make it the name more general
```suggestion
const TextareaInput = () => {
```